saleor/graphql/product/mutations/product/product_create.py

```diff
@@ -220,7 +219,8 @@ def _save_m2m(cls, _info: ResolveInfo, instance, cleaned_data):
     @classmethod
     def post_save_action(cls, info: ResolveInfo, instance, _cleaned_input):
         product = models.Product.objects.prefetched_for_webhook().get(pk=instance.pk)
-        update_product_search_vector(instance)
+        product.search_index_dirty = True
+        product.save(update_fields=["search_index_dirty"])
```

Comment:
For the product mutations we already run the `save` function above. It should be enough to add:

```
product.search_index_dirty = True
```

in line 208 before `instance.save()` happens.